operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -289,20 +332,15 @@ This procedure will add a liquid-cooled blades from a HPE Cray EX system.
     ```
 
 #### Check firmware
+17.   Verify that the correct firmware versions for node BIOS, node controller (nC), NIC mezzanine card (NMC), GPUs, and so on.
```

Comment:
The spacing/indentation of the sub-steps is not formatted correctly, causing the rendered version to display incorrectly.